frontend/src/utils/utils.ts

```diff
@@ -244,3 +244,20 @@ export function setEmptyStringsToNull(data: any) {
 
   return data;
 }
+
+/**
+ * @function concatAddress
+ * turns different location data pieces into one address string
+ * @param {object} data address values to be concatenated
+ * @returns {String} concatenated address string
+ */
+export function concatenateAddress(data: any) {
+  const locationAddress = [];
+  if (data.streetAddress?.length) locationAddress.push(data.streetAddress);
+  if (data.locality?.length) locationAddress.push(data.locality);
+  if (data.province?.length) locationAddress.push(data.province);
+
+  const locationAddressStr = locationAddress.join(', ');
+
+  return locationAddressStr;
+}
```

Comment:
Rather than having the util function require a data object with the three fields, it might be better to have it instead require the function to receive 3 separate (and optional) arguments. This would allow for better reuse elsewhere.